timeless/views.py

```diff
@@ -224,6 +225,51 @@ def get_default_context(self):
         return context
 
 
+class CreateView(GenericView):
+    """ Class which creates objects based on received POST data and provided
+    form class """
+    form_class = None
+    success_view_name = None
```

Comment:
@vryazanov Hmm, I think it's actually not view name it's url name. wdyt?

Reply:
@vladarefiev I thought about it, but then decided that view name is better then hardcode url